code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -297,21 +297,26 @@
 	gadpathur["medical cadre brassard"] = /obj/item/clothing/accessory/armband/gadpathur/med
 	gear_tweaks += new/datum/gear_tweak/path(gadpathur)
 
-/datum/gear/accessory/dominia/sash
-	display_name = "dominian house sash selection"
-	description = "A selection of sashes representing the Great Houses of Dominia."
-	path = /obj/item/clothing/accessory/dominia/sash
+/datum/gear/accessory/sash_coloured
+	display_name = "sash (colourable)"
+	path = /obj/item/clothing/accessory/sash/colourable
```

Comment:
bri'ish detected

Reply:
this is consistent with the colourable armband :(